### PR TITLE
en,zh: Bump tidb components to v7.1.0

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

```bash
find . -type f | xargs sed -i "s/v6.5.0/v7.1.0/g"
git checkout -- zh/releases en/releases
```

**and with some manually fix**

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/get-started.md [hunk 1]
```diff
@@ -496,10 +496,10 @@ APPROXIMATE_KEYS: 0
 ```sql
 mysql> select tidb_version()\G
 *************************** 1. row ***************************
-  tidb_version(): Release Version: v6.5.0
+  tidb_version(): Release Version: v7.1.0
          Edition: Community
  Git Commit Hash: 4a1b2e9fe5b5afb1068c56de47adb07098d768d6
-      Git Branch: heads/refs/tags/v6.5.0
+      Git Branch: heads/refs/tags/v7.1.0
```
Comment (csuzhangxc) [inline en/get-started.md:502]: it's better to update the above `Git Commit Hash` to the real value after TiDB v7.1.0 is released.

File: zh/deploy-on-gcp-gke.md [hunk 2]
```diff
@@ -270,7 +270,7 @@ gcloud compute instances create bastion \
     $ mysql --comments -h 10.128.15.243 -P 4000 -u root
     Welcome to the MariaDB monitor.  Commands end with ; or \g.
     Your MySQL connection id is 7823
-    Server version: 5.7.25-TiDB-v6.5.0 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
+    Server version: 5.7.25-TiDB-v7.1.0 TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible
```
Comment (csuzhangxc) [inline zh/deploy-on-gcp-gke.md:273]: the part of `5.7.25` may be changed and it's better to update it after TiDB v7.1.0 is released.

File: en/get-started.md (hunk 1 above)
Comment (ran-huang) [reply en/get-started.md:502]: Can you please update this now that 7.1 is released? Thank you~

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**